### ISSUE TITLE
Warp CTC version in Makefile

### ISSUE THREAD

**Issue by miguelvr** [espnet/espnet#290]
Since I couldn't run with multi GPU with pytorch 0.4.0, I decided to downgrade it to 0.3.1. However, I noticed a substantial slowdown in training. I think one factor might be the commit tag we are using in the Makefile, in which the GPU binding don't use the flag `voltile=True` (0.3.X) neither `with torch.no_grad()`, leading to the gradients being unnecessarily recorded in the CTC... 

I might be wrong, but we should at least change to this [commit](https://github.com/SeanNaren/warp-ctc/commit/34ab2673b8c5b99d8fc52bb04d53cfebbfc24fa3) for improved performance with v0.3.1. 

This, however, is not a solution for the problem in version 0.4, so we should consider alternatives if backward campatibility is something desired.

Best regards,
Miguel

**Comment by sw005320** [comment]
Thanks. I'll test it.

**Comment by kan-bayashi** [comment]
In current master, this problem is fixed.
I close this issue.
